src/integration/java/io/pinecone/integration/dataplane/UpdateAndQueryTest.java

```diff
@@ -56,6 +57,7 @@ public void UpdateRequiredParamsFetchAndQuerySync() throws InterruptedException
                 .build();
 
         blockingStub.update(updateRequest);
+        Thread.sleep(3500);
```

Comment:
When I switch to data plane refactoring, the PR will update thread.sleep with assertWithRetry.